addons/sys_core/XEH_postInit.sqf

```diff
@@ -151,4 +150,37 @@ if (getClientStateNumber < 10) then { // Check before game has started (in brief
     ADDPFH(_briefingCheck, 0, []);
 };
 
+///////////////////////////////////
+//
+// CBA SETTINGS
+// Applies the difficulty module settings over CBA settings. This allows the mission editor to bypass cba server settings.
+// If the module is not present, this function has no effect.
+//////////////////////////////////
+{
+    private _ignoreAntennaDirection = ((allMissionObjects "acre_api_DifficultySettings") select 0) getVariable ["IgnoreAntennaDirection", nil];
```

Comment:
I would suggest storing the result of -> `(allMissionObjects "acre_api_DifficultySettings")` as it's a relatively expensive command, so doing it just once would be great. Also would be good to handle the case where there aren't any modules!